var/spack/repos/builtin/packages/opensubdiv/package.py

```diff
@@ -37,6 +39,7 @@ def url_for_version(self, version):
     depends_on("glfw@3.0.0:")
     depends_on("intel-tbb@4.0:", when="+tbb")
     depends_on("libxrandr")
+    depends_on("libxxf86vm")
```

Comment:
Is the a dependency for all of the versions?  Or just those since `3.5`?

If the latter, then consider changing to a conditional dependence (see docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-dependencies).

Reply:
It is on at least 3.4.3. I don't think I tried 3.4.0.

Speaking of 3.4.3 (and presumably 3.4.0), it's broken. After I fixed the missing library, it said...
```
1200      File "/path/to/spack-stage/spack-stage-opensubdiv-3.4.3-o3ifjr2nrpr5oyzdq3h6qfb62jqiwcmu/spack-src/documentation/processTutorials.py", line 36
1201        print "Could not read file \'"+inputfile+"\'"
1202        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
1203    SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?
```
... which I _think_ means the 3.4.x branch is still on Python 2. I'm sure there's a way to have Spack take care of it, but I don't know what that way is and I figured since it's a separate issue it should probably be its own PR anyway.

Reply:
Spack no longer supports Python 2, which was deprecated in v0.19 and removed in v0.20.

How much flexibility is there in that build?  Is there an option that could be used to disable those parts of the build (see https://github.com/spack/spack/pull/34851/files for an example).

Or maybe just deprecate that version (https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions) with a comment explaining why.